### ISSUE TITLE
"filtlong" module broke

### ISSUE THREAD

**Issue by richardstoeckl** [ewels/MultiQC#1843]
### Description of bug

Hi, thank you very much for this incredible software!

The filtlong module seems to have a problem with the parsing of the total amount of reads in the attached log file:

`ValueError: could not convert string to float: '379.964.567'`

MultiQC is called via the ["onsuccess"](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#onstart-onsuccess-and-onerror-handlers) handler of a [Snakemake](https://snakemake.readthedocs.io/en/stable/index.html) pipeline with the following command: 

```
onsuccess:
    print("[|87 Workflow finished without errors!")
    shell("multiqc -f results")
```

Running on Ubuntu 20.04 LTS, installed via conda.

### File that triggers the error

[BBR140205_filtlongCombined.log](https://github.com/ewels/MultiQC/files/10435116/BBR140205_filtlongCombined.log)

### MultiQC Error log

```console
/// MultiQC 🔍 | v1.14

|           multiqc | Search path : /home/rks/phd/automatedassembly/results
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1256/1256  
|             fastp | Found 18 reports
╭──────────────────────────────────────── Oops! The 'filtlong' MultiQC module broke... ────────────────────────────────────────╮
│ Please copy this log and report it at https://github.com/ewels/MultiQC/issues                                                │
│ Please attach a file that triggers the error. The last file found was: results/BBR140205/logs/BBR140205_filtlongCombined.log │
│                                                                                                                              │
│ Traceback (most recent call last):                                                                                           │
│   File "/home/rks/miniconda3/lib/python3.9/site-packages/multiqc/multiqc.py", line 654, in run                               │
│     output = mod()                                                                                                           │
│   File "/home/rks/miniconda3/lib/python3.9/site-packages/multiqc/modules/filtlong/filtlong.py", line 26, in __init__         │
│     self.parse_logs(f)                                                                                                       │
│   File "/home/rks/miniconda3/lib/python3.9/site-packages/multiqc/modules/filtlong/filtlong.py", line 47, in parse_logs       │
│     self.filtlong_data[f["s_name"]] = {"Target bases": float(l.lstrip().split(" ")[1])}                                      │
│ ValueError: could not convert string to float: '379.964.567'                                                                 │
│                                                                                                                              │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
|            pycoqc | Found 2 reports
|          minionqc | Found 2 reports
|           multiqc | Compressing plot data
|           multiqc | Deleting    : multiqc_report.html   (-f was specified)
|           multiqc | Deleting    : multiqc_data   (-f was specified)
|           multiqc | Report      : multiqc_report.html
|           multiqc | Data        : multiqc_data
|           multiqc | MultiQC complete
CalledProcessError in file /home/rks/phd/automatedassembly/Snakefile, line 473:
Command 'set -euo pipefail;  multiqc -f results' returned non-zero exit status 1.
  File "/home/rks/phd/automatedassembly/Snakefile", line 473, in __onsuccess
```
```

**Comment by ewels** [comment]
Oh no, another internationalisation error. It's because filtlong is using the locale-specific numeric style of `.` for thousand separators. These problems only really happen for Germans / Swedes / people from other European countries that use this format. It's one of my least favourite ones to fix 😰 

Here I guess it should be ok, as we will never have a decimal count, so we can just strip all `.` characters from the string before parsing.

**Comment by ewels** [comment]
Ok, should be fixed in 82805e63 - thank you for reporting!
